yarn-project/simulator/src/avm/avm_memory_types.test.ts

```diff
@@ -167,21 +167,6 @@ describe('Field', () => {
     expect(result).toStrictEqual(new Field(2n));
   });
 
-  it(`Should check equality of two Fields correctly`, () => {
-    const field1 = new Field(5);
-    const field2 = new Field(5);
-    const field3 = new Field(10);
-    expect(field1.equals(field2)).toBe(true);
-    expect(field1.equals(field3)).toBe(false);
-  });
```

Comment:
EQ should support be supported for fields, but LT and LTE should not. We are basically copying Brillig here:
![image](https://github.com/AztecProtocol/aztec-packages/assets/47112877/8ba2815e-6df5-44a8-837d-0566d5b5eab0)

https://github.com/AztecProtocol/aztec-packages/blob/master/noir/acvm-repo/brillig/src/opcodes.rs#L173